.gitignore

```diff
@@ -25,8 +25,12 @@ pyrealm.egg-info
 # Coverage
 .coverage
 
+# mypy
+.mypy_cache/
+
 # Data
-pyrealm_build_data/*.nc
+pyrealm_build_data/inputs_data_24.25.nc
+pyrealm_build_data/eda.py
```

Comment:
Is `eda.py` the subsetting script? I'm not quite sure what the best thing to do here is. In an ideal world, we'd be able to regenerate the input file completely - but I'd need to add all the download scripting to assemble the file we were using. Ultimately, I think that is where an artificial input would be cleaner.

Reply:
Yes, that's for excluding my local script to analyze and subset the dataset. I can write a bash script to regenerate the input file from the LFS file if it's better.